vendor/modules.txt

```diff
@@ -108,6 +108,8 @@ github.com/kisielk/gotool/internal/load
 github.com/konsorten/go-windows-terminal-sequences
 # github.com/magiconair/properties v1.8.1
 github.com/magiconair/properties
+# github.com/maratori/testpackage v1.0.0
```

Comment:
Please rebase and remove vendoring :slightly_smiling_face:

Reply:
Done